### ISSUE TITLE
Aurora UI rendering stutters - causes other windows stuttering until minimized

### ISSUE THREAD

**Issue by olokos** [antonpup/Aurora#1403]
### Expected behavior:
Smooth movements of Aurora UI

### Actual behavior:
Stuttering UI, stutter when moving the window itself, causes other windows and even mouse cursor to stutter aswell, just having it opened and in focus.
### How often does this problem occur?

This problem occurs every time

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. Open Aurora
2. Move the window
3. Escalate the problem by forcing to refresh Aurora UI even when out of focus

### What is the version of Aurora you are experiencing this issue with?

My version is... 0.6.3-537

### Does the previous version of Aurora have this issue?
All the way down to -532, I'm pretty sure since first time that I used it it had this problem.

### Please include any logs, screenshots, or files that are related to this issue:

[Video showing the issue](https://www.youtube.com/watch?v=bBfYnymEfR0)
It's more visible on actual PC than on video.
I'm using two monitors Main 144Hz G-Sync and second 60Hz, both 1080p.

**Comment by simon-wh** [comment]
I have a feeling this may be just a problem with WPF (the UI framework Aurora uses), I don't know if there's a whole pile we can do about it. From the video I only really noticed a slight bit of stuttering as the window was being moved about, is that it?

**Comment by simon-wh** [comment]
Hmmm, I have a feeling I know where this problem is originating, I'll try and have a look into it to see what I can do.

**Comment by olokos** [comment]
If I got it correctly we would need to rewrite functions that are using dispatcher to another ones?

**Comment by simon-wh** [comment]
Yeah, the way it's setup right now isn't great, but it's all going to be changed soon anyway so I'll try and sort it out then.
